packages/reactive-element/src/decorators/base.ts

```diff
@@ -23,15 +23,15 @@ export interface ClassElement {
   kind: 'field' | 'method';
   key: PropertyKey;
   placement: 'static' | 'prototype' | 'own';
-  initializer?: Function;
+  initializer?: () => void;
   extras?: ClassElement[];
   finisher?: <T>(clazz: Constructor<T>) => void | Constructor<T>;
   descriptor?: PropertyDescriptor;
 }
 
 export const legacyPrototypeMethod = (
   descriptor: PropertyDescriptor,
-  proto: Object,
+  proto: unknown,
```

Comment:
proto should at least be `object` yeah? i.e. it can't be null, undefined, or a primitive